.buildkite/pipeline_perf.py

```diff
@@ -119,7 +119,10 @@
 # }
 # will pin steps running on instances "m6i.metal" with kernel version tagged "linux_6.1"
 # to a new kernel version tagged "linux_6.1-pinned"
-pins = {}
+pins = {
+    "linux_5.10-pinned": {"instance": "m6i.metal", "kv": "linux_5.10"},
+    "linux_6.1-pinned": {"instance": "m6i.metal", "kv": "linux_6.1"},
+}
```

Comment:
This would only pin them for this test. If we want to apply it to all tests, we would have to change it in `common.py`

Reply:
Let's do that if we see other tests are affected, thanks!